src/main/kotlin/org/elaastic/questions/assignment/sequence/SequenceService.kt

```diff
@@ -81,6 +82,14 @@ class SequenceService(
         } ?: throw EntityNotFoundException("There is no sequence for id \"$id\"")
     }
 
+    // TODO: delete this method!
+    @Deprecated("Will be deleted",
```

Comment:
Humm, is it normal to see a method to be deleted, added?
A rebase issue?

Reply:
Yes, it's a rebase issue.

Fix in 8452777041ef5c91bfbe746ac4612636f48212df